radio/src/targets/common/arm/stm32/f0/system_stm32f0xx.c

```diff
@@ -158,6 +161,33 @@ static void SetSysClock(void);
   */
 void SystemInit (void)
 {    
+
+  // Check if we should go into bootloader mode.
+  //
+  // Set the main stack pointer __set_MSP() to its default value.  The default
+  // value of the main stack pointer is found by looking at the default value
+  // in the System Memory start address. Do this in IAR View -> Memory.  I
+  // tried this and it showed address: 0x200014A8 which I then tried here.
+  // The IAR compiler complained that it was out of range.  After some
+  // research, I found the following from "The STM32 Cortex-M0 Programming
+  // Manual":
+  //         Main Stack Pointer (MSP)(reset value). On reset, the processor
+  //         loads the MSP with the value from address 0x00000000.
+  //
+  // So I then looked at the default value at address 0x0 and it was 0x20002250
+  //
+  // Note that 0x1fffC800 is "System Memory" start address for STM32 F0xx
+  //
+  if ( *((unsigned long *)0x20003FF0) == 0xDEADBEEF ) {
```

Comment:
I'm correctly entering this section after reset, but it seems I'm not jumping to the correct location.

Reply:
Yes it's just a trigger, i had the same issue after jump.